packages/Checkbox/stories/Checkbox.stories.js

```diff
@@ -1,28 +1,34 @@
 import React from "react";
-import { storiesOf } from "@storybook/react";
 import { withKnobs } from "@storybook/addon-knobs";
-import { Story } from "storybook/assets/styles/common.styles";
+import { showcaseStoryParameters, variationsStoryParameters } from "storybook/assets/storyParameters";
 import { getStoryName } from "storybook/storyTree";
-import CheckboxAlign from "./examples/CheckboxAlign";
+import ExampleStory from "storybook/components/ExampleStory";
 import Variations from "./examples/Variations";
-import CheckboxGrouping from "./examples/CheckboxGrouping";
 import ShowcaseStory from "./examples/Showcase";
+import Checkbox from "../src";
 
-const storyName = getStoryName("Checkbox");
+export default {
+  title: getStoryName("Checkbox"),
+  component: Checkbox,
```

Comment:
Maybe don't need this

Reply:
When I removed this part, the showcase is gone from the browser.. so probably we need this?